### PR TITLE
[DS-1545] Add a profile name to the 'core.extension.module' list

### PR DESCRIPTION
**Pull request by aleevas**
Original Issue, this PR is going to fix: [DS-1545](https://yusa.atlassian.net/browse/DS-1545)

Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature or this is a bug fix - use `main` branch. We'll tag for release if the bug is critical ASAP or tag for release next bug fix release until the critical issue arrives.

## Steps for review

- [ ] log as admin
- [ ] go to /admin/reports/status
- [ ] verify that the Drupal Core version is 10.3.10
image

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] Try to use Conventional commit messages accordingly to the standard https://www.conventionalcommits.org/en/v1.0.0/#specification
- [ ] [Documentation](https://github.com/ycloudyusa/yusaopeny/tree/main/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ycloudyusa/yusaopeny/blob/main/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with an account on drupal.org, otherwise, you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ycloudyusa/yusaopeny/blob/main/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

File: openy.profile [hunk 1]
```diff
@@ -800,3 +804,22 @@ function openy_enable_search_api_solr_legacy() {
...
+/**
+ * The profile name should exist in the 'core.extension.module' list.
+ *
+ * @see https://git.drupalcode.org/project/drupal/-/commit/bd13fb1a9091996e526110851c73137a01e78cb3#a96e18f6425fb567a1083b3c6f8afcfaa94b8522_133_134
+ */
+function openy_register_profile(): void {
+  $config_factory = Drupal::configFactory();
+  $extension_config = $config_factory->getEditable('core.extension');
+  // Set the default and admin theme.
+  $config_factory
```
Comment (aleevas) [inline openy.profile:817]: I've added these configurations here because they are in the profile.install, they didn't work

**Comment by rollki** [comment]
test this please

**Comment by podarok** [comment]
> test this please

builder is in upgrade to php 8.3 state
could be failing, not working...

**Comment by rollki** [comment]
It looks like it worked.
Screenshot 2024-12-27 at 14 13 03

**Comment by podarok** [comment]
builds are not working correctly from forks

**Pull request merged**